src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#logo">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#apple" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#samsung" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+            <a href="#smartphones" class="nav__link">Smartphones</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+            <a href="#Laptops & computers" class="nav__link">Laptops & computers</a>
```

Comment:
you shouldn't use spaces for your href attribute and instead of symbols like `&` you can use encoded value `%26`
```suggestion
            <a href="#laptops%26computers" class="nav__link">Laptops & computers</a>
```